crates/node/ethstats/src/error.rs

```diff
@@ -0,0 +1,69 @@
+use thiserror::Error;
+
+/// Errors that can occur during `WebSocket` connection handling
+#[derive(Debug, Error)]
+pub enum ConnectionError {
```

Comment:
I think since this more like an application, we could also just use `eyre::Result` for error handling.

but now that we have this, we can keep